test/src/shouldFail.test.js

```diff
@@ -51,6 +55,20 @@ describe('shouldFail', function () {
     it('rejects an outOfGas', async function () {
       await assertFailure(shouldFail.reverting(this.failer.failWithOutOfGas({ gas: 2000000 })));
     });
+
+    describe('reverting.withMessage', function () {
+      it('rejects if no failure occurs', async function () {
+        await assertFailure(shouldFail.reverting.withMessage(this.failer.dontFail()));
```

Comment:
Can we also add a test case where a revert _does_ occur, but the message is wrong?